portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -18,6 +18,11 @@
 	<td>//div[contains(@class,'ddm-field')]//button[contains(@class,'ddm-form-field-repeatable-add-button')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>FIELD_REPEATABLE_BUTTOM</td>
+	<td>//div[contains(@class,'ddm-field')]//div[contains(@data-field-name,'${key_fieldName}')]//button[contains(@class,'ddm-form-field-repeatable-add-button')]</td>
```

Comment:
@leticiachiu I just have one question, how do you distinguish between the two fields since the `key_fieldName` is the same?

Reply:
fixed :D